edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Constants.cs

```diff
@@ -43,6 +43,8 @@ public static class Constants
 
         public const string EdgeModuleHubServerCertificateFileKey = "EdgeModuleHubServerCertificateFile";
 
+        public const string CheckImagePullBeforeModuleCreate = "CheckImagePullBeforeModuleCreate";
```

Comment:
@veyalla - we added a flag for the EdgeAgent change (just in case). Any suggestions on the naming?

Reply:
I'll wait a few hours for his input.